Forged_Feedback/README.md

```diff
@@ -0,0 +1,38 @@
+# Forged Feedback
+
+## Type
+
+Broken Access Control
+
+## Description
+
+Post some feedback in another user's name.

```

Comment:
answer the questions how and what in the description part.

Reply:
its the description of challenge in Juice Shop too